src/prompts/personas2.ts

```diff
@@ -0,0 +1,68 @@
+type Persona = {
+  /** Role name including seniority */
+  roleName: string;
+  /** Longer form, describing their characteristics. weaknesses, strengths, etc */
+  description: string;
+  /**
+  The strategic development objectives focus on harnessing a deep comprehension of the codebase architecture and dependencies, coupled with an emphasis on coding efficiency and quality. These objectives aim to foster a culture of innovation and optimization, where opportunities for enhancing the codebase are actively pursued. Mastery in development workflow, including adherence to coding standards and leveraging automated testing, is prioritized to streamline processes. Additionally, identifying and mitigating performance bottlenecks, along with fortifying the security of code and data, are critical goals to ensure the robustness and integrity of applications.
+  */
+  developmentObjectives: Array<string>;
+}
+
+const personas: Array<Persona> = [
```

Comment:
@kyegomez : Trying to generate prompts based on templates with a variety of options to increase chance of diverse replies/scenarios.

Initially focused on the question generation right now. Refining those prompts.

Reply:
Structure can be something different, too. Be as creative as you like with the template(s).

For example, this file could be seeds for 1 prompt template.

A swarm to come up with a lot of *diverse* and *realistic* prompts would be great!